docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,305 @@
+
+* start_display: The readably formatted start date of the course.
+
+* start_type: Indicates how `start_display` is set. Possible values are:
+
+  * "string": manually set
+  * "timestamp": generated form `start` timestamp
+  * "empty": the start date should not be shown
+
+* pacing: The type of course pacing for this course. Possible values:
+  instructor, self
+
+* course_id: Course key. This field might be returned but is deprecated. You
+  should use ``id`` instead.
+
+* course_id: Course key. This field might be returned but is deprecated. You
+  should use ``id`` instead.
+
+================
+Return Values
+================
+
+* 200 on success with above fields.
+
+* 400 if an invalid parameter was sent, or if the username was not provided
+  for an authenticated request.
+
+* 403 if a user who does not have permission to masquerade as another user
+  specifies a username other than their own.
+
+* 404 if the course is not available or cannot be seen.
+
+=========================================================
+Example Response Showing Details of a Specified Course
+=========================================================
+
+::
+
+ {
+    "blocks_url": "https://courses.edx.org/api/courses/v1/blocks/?course_id=edX%2FDemoX.1%2F2014",
+    "effort": null,
+    "end": "2030-12-31T00:00:00Z",
+    "enrollment_start": null,
+    "enrollment_end": null,
+    "id": "edX/DemoX.1/2014",
+    "media": {
+        "course_image": {
+            "uri": "/c4x/edX/DemoX.1/asset/edX101MarketingContent_DashboardImage.png"
+        },
+        "course_video": {
+            "uri": null
+        },
+        "image": {
+            "raw": "https://d37djvu3ytnwxt.cloudfront.net/c4x/edX/DemoX.1/asset/edX101MarketingContent_DashboardImage.png",
+            "small": "https://d37djvu3ytnwxt.cloudfront.net/c4x/edX/DemoX.1/thumbnail/edX101MarketingContent_DashboardImage-png-375x200.jpg",
+            "large": "https://d37djvu3ytnwxt.cloudfront.net/c4x/edX/DemoX.1/thumbnail/edX101MarketingContent_DashboardImage-png-750x400.jpg"
+        }
+    },
+    "name": "Demo Course",
+    "number": "DemoX.1",
+    "org": "edX",
+    "short_description": "",
+    "start": "2014-08-01T04:00:00Z",
+    "start_display": "Aug. 1, 2014",
+    "start_type": "timestamp",
+    "pacing": "instructor",
+    "overview": "<p>Include your long course description here.</p>"
+ }
+
+
+.. _Get the list of courses visible to a user:
+
+*****************************************
+Get a List of Courses Visible To a User
+*****************************************
+
+The endpoint to get a list of courses that are visible to a specified user is
+``/api/courses/v1/courses/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of courses that are visible to a specified user.
+If the request is made by an anonymous user, a username is not required.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
```

Comment:
Maybe the example requests should show a complete URL? One common way to show example requests is with a curl command like this:

.. code-block:: bash

  curl -X GET \
  -H "Authorization: Bearer 6ec0b691bd145248b5d5c66e5fc587d983ffd3c5" \
  http://myopenedxhost.org/api/courses/v1/courses/